### PR TITLE
Ip 4971

### PR DESCRIPTION
**Pull request by redraiderg**

### CODE REVIEW

**Review by jsnwgnr — COMMENTED** [review]
Should all these AVL options be set to true?? AVALANCHE_FEATURE_VISIBLE_ENV

**Comment by redraiderg** [comment]
> 



> Should all these AVL options be set to true?? AVALANCHE_FEATURE_VISIBLE_ENV

That's how they were before, but I can modify if desired.  I'll ask Robert

File: templates/static-content-configmap.yaml [hunk 1]
```diff
@@ -33,7 +33,113 @@ data:
       "BASE_URL_MACRO": "{{ .Values.staticContent.baseEndpoint }}/api",
       "BASE_URL_DESTINATION": "{{ .Values.staticContent.baseEndpoint }}/api"
     }
-
+  aopAppConfig.json: |
```
Comment (mgabriel35) [inline templates/static-content-configmap.yaml:36]: I would say go ahead and make this a single value in the helm chart and put this file in the values file. Makes it easier to change in the future.

File: templates/static-content-configmap.yaml [hunk 2]
```diff
@@ -33,7 +33,8 @@ data:
...
+    {{ required "config.aopAppConfig is required " .Values.staticContent.aopAppConfig | toString | indent 4 | trim  }}
```
Comment (auswells) [inline templates/static-content-configmap.yaml]:
```suggestion
    {{ required "staticContent.aopAppConfig is required " .Values.staticContent.aopAppConfig | toString | indent 4 | trim  }}
```

**Review by auswells — APPROVED** [review]

**Review by mgabriel35 — APPROVED** [review]

File: templates/static-content-configmap.yaml (hunk 1 above)
Comment (redraiderg) [reply templates/static-content-configmap.yaml:36]: That's what Austin suggested as well. Should be in there now.

**Pull request merged**